web/template/payments.gohtml

```diff
@@ -0,0 +1,58 @@
+{{ template "page" . }}
+
+{{ define "title" }}Payments{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <p class="govuk-body"><strong>{{ if eq .Case.SubType "pfa" }}PFA{{ else if eq .Case.SubType "lpa" }}LPA{{ end }} {{ .Case.UID }}</strong></p>
+            <p class="govuk-body"><strong>Total paid: £{{ fee .TotalPaid }}</strong></p>
+
+            {{ range $i, $p := .Payments }}
+                <details class="govuk-details" data-module="govuk-details" open>
+                    <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text">
+                  Payment {{ plus1 $i }}
+                </span>
+                    </summary>
```

Comment:
Not a serious issue, but the indenting is a bit confused here.

Reply:
My bad, I think its because I copied the details example from GDS